### PR TITLE
Feature/skill list page

### PR DESCRIPTION
**Pull request by Denyer96**
Please have a look at the session window page

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import Page1 from './Page1'
 import Session from './Session'
 import Review from './Review'
+import SkillList from './SkillList'
```
Comment (TechDevProNBS) [inline src/components/Home.js]: You don't need this import here

File: src/components/SessionWindow.js [hunk 2]
```diff
@@ -17,10 +17,19 @@ export default class SessionWindow extends React.Component {
 
     this.state = {
       remindInDate: 0,
-      customGoals: []
+      customGoals: [],
+      newSkillList: []
     };
   }
-
+  handleForm = newArray => {
```
Comment (TechDevProNBS) [inline src/components/SessionWindow.js:25]: You haven't added documentation

File: src/components/SkillListTest.js [hunk 3]
```diff
@@ -0,0 +1,176 @@
+import React from "react";
```
Comment (TechDevProNBS) [inline src/components/SkillListTest.js]: This doesn't look like you're testing the skillList.js page. Do we need this file?

**Review by TechDevProNBS — CHANGES_REQUESTED** [review]
Document your work please!

File: src/components/SkillList.js [hunk 4]
```diff
@@ -143,11 +143,13 @@ export default class SkillList extends React.Component {
             array: []
         }
     }
+    //calls the handleForm function on session window
     handleForm() {
         var newArray = this.state.array
         alert("hello")
         this.props.handleForm(newArray)
       };
+    //creates and array and also deletes entries if needed
```
Comment (TechDevProNBS) [inline src/components/SkillList.js]: Correct grammer

**Review by TechDevProNBS — CHANGES_REQUESTED** [review]

**Review by JameswDemps — APPROVED** [review]

**Review by TechDevProNBS — APPROVED** [review]

**Pull request merged**